README.ui-testing.md

```diff
@@ -0,0 +1,91 @@
+# Ghost Inspector Testing
+
+Run [Ghost Inspector](http://ghostinspector.com/) tests on your `mh-opsworks` cluster using the `./bin/mh_ghost` script. This script acts as a wrapper around the ghost inspector commands provided by the
```

Comment:
Is there any way to package up a set of ghost inspector tests so that another organization could start with our base tests?